### ISSUE TITLE
no-spec-dupes false positive when nesting describes

### ISSUE THREAD

**Issue by BenjaminVanRyseghem** [tlvince/eslint-plugin-jasmine#167]
**Do you want to request a *feature* or report a *bug*?**

a bug

**What is the current behavior?**

When describes are nested, two inner describe with the same name fails.

**If the current behavior is a bug, please provide the steps to reproduce.**

```js
describe("foo", () => {
    describe("baz", () => {})
}

describe("bar", () => {
    describe("baz", () => {}) // eslint error because "baz" is duplicated
}
```

**What is the expected behavior?**

It should detect that foo.baz and bar.baz are different.

**Please mention your node.js, eslint-plugin-jasmine and operating system version.**

node --version
v10.6.0

"name": "eslint-plugin-jasmine",
"version": "2.10.1"

operating system
macOS High Sierra 10.13.4 (17E202)

**Comment by BenjaminVanRyseghem** [comment]
same goes with `no-spec-dupes`.

Same spec name in two different suites fails
